src/css/main.css

```diff
@@ -1,113 +1,93 @@
 body {
-    margin: 0;
-    padding: 0;
-    font-family: sans-serif;
-}
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif; }
+
 .MainHeader {
-    position: absolute;
-    left: 50%;
-    top: 5%;
-    transform: translate(-50%, -50%);
-    color: white;
-    font-size: 20px;
-}
+  position: absolute;
+  left: 50%;
+  top: 5%;
+  transform: translate(-50%, -50%);
+  color: white;
+  font-size: 12px; }
 
 @media screen and (min-height: 400px) {
-    .MainHeader {
-        font-size: 30px;
-    }
-}
+  .MainHeader {
+    font-size: 32px; } }
 
 .BelowTitle {
-    position: absolute;
-    left: 20%;
-    top: 10%;
-    width: 80%;
-}
+  position: absolute;
+  left: 20%;
+  top: 10%;
+  width: 80%; }
 
 .OnlineUsers {
-    position: absolute;
-    top: 20%;
-    right: 30%;
-    padding-left: 8px;
-}
+  position: absolute;
```

Comment:
I tried changed position on OnlineUsers to fixed and it stopped the Online Users box kinda getting lower and lower the more messages there were - just kept it in the same position on the page basically. See if it works for you / whether I'm just failing at frontend

Reply:
Hmmm, it's mostly all just placeholder CSS for now. Going to rewrite it anyway.